### PR TITLE
Logging for endpoints

### PR DESCRIPTION
**Pull request by trevlenb2**

### CODE REVIEW

File: src/main/java/com/revealprecision/revealserver/config/CustomRequestBodyAdviceAdapter.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.sleuth.Tracer;
+import org.springframework.http.HttpInputMessage;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.servlet.mvc.method.annotation.RequestBodyAdviceAdapter;
+
+@ControllerAdvice
+@Slf4j
+public class CustomRequestBodyAdviceAdapter extends RequestBodyAdviceAdapter {
```
Comment (milank7) [inline src/main/java/com/revealprecision/revealserver/config/CustomRequestBodyAdviceAdapter.java:24]: Can we do dependency injection using Lombok same as in the other places?

File: src/main/java/com/revealprecision/revealserver/config/CustomResponseBodyAdviceAdapter.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import org.springframework.http.server.ServerHttpResponse;
+import org.springframework.http.server.ServletServerHttpRequest;
+import org.springframework.http.server.ServletServerHttpResponse;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseBodyAdvice;
+
+
+@ControllerAdvice
+@Slf4j
+public class CustomResponseBodyAdviceAdapter implements ResponseBodyAdvice<Object> {
```
Comment (milank7) [inline src/main/java/com/revealprecision/revealserver/config/CustomResponseBodyAdviceAdapter.java:28]: Can we do dependency injection using Lombok same as in the other places?

**Review by milank7 — COMMENTED** [review]
Looks good, take a look in 2 comments

**Comment by lindagcaba** [comment]
@trevlenb2 please update, so we can merge

**Comment by lindagcaba** [comment]
@milank7 as reviewer, please check updates, if happy you can merge.

**Review by milank7 — APPROVED** [review]

**Pull request merged**